docker/docker-compose.test.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3.8"
```

Comment:
It seems a bit smelly that we'd need a separate `docker-compose` configuration exclusively for running tests, maybe it's worth documenting the reasoning here in a comment?